tests/unit/test_asynchronous.py

```diff
@@ -1,5 +1,6 @@
 import os
 from unittest.mock import MagicMock, patch, AsyncMock
+import unittest
```

Comment:
We use `pytest` instead of `unittest`, please remove all references to it.

Reply:
Fixed